site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -270,20 +244,19 @@ extensionPanel.onShown.addListener(function (extPanelWindow) {
 });
 ```
 
-### Messaging from content scripts to the DevTools page {: #content-script-to-devtools }
+### Send messages between content scripts and the DevTools page {: #content-script-to-devtools }
 
-Messaging between the DevTools page and content scripts is indirect, by way of the service worker.
+Use the service worker to send messages between the DevTools page and content scripts.
 
-When sending a message _to_ a content script, the service worker can use the
+To send a message _to_ a content script, the service worker can use the
 [`tabs.sendMessage()`][api-scripting-sendmessage] method, which directs a message to the content
```

Comment:
```suggestion
[`tabs.sendMessage()`][api-scripting-sendmessage], which directs a message to the content
```

Reply:
Every other place on this page that introduces a method uses the "the [name] method" phrasing. What makes this one different?